src/arch/arm/fpu/vfp.h

```diff
@@ -0,0 +1,68 @@
+/**
+ * @file vfp.h
+ * @brief
+ * @author Denis Deryugin <deryugin.denis@gmail.com>
+ * @version
+ * @date 28.03.2018
+ */
+
+#ifndef ARM_FPU_VFP_H_
+#define ARM_FPU_VFP_H_
+
+#define FPU_DATA_LEN 65 /* One word for coprocessor state
+			   and 64 words for VPF registers */
+
+#ifndef __ASSEMBLER__
+#include <string.h>
+static inline void arm_fpu_context_init(void *opaque) {
+	memset(opaque, 0, sizeof(uint32_t) * FPU_DATA_LEN);
```

Comment:
It's really  necessary ?

Reply:
Well, probably this can be removed, but I think it's ok to make sure there's no garbage in this memory.